### ISSUE TITLE
Focal lupus nephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6834]
**Preferred term label:**
Focal lupus nephritis

**Synonyms**
Class III lupus nephritis

**Definition (free text, please give PubMed ID)**
A subclass of lupus nephritis with active or inactive focal, segmental or global endocapillary or extracapillary glomerulonephritis involving <50% of all glomeruli, typically with focal subendothelial immune deposits, with or without mesangial alterations 
PMID: 31974366

**Parent term (use hpo.jax.org/app)**
Lupus nephritis (newly proposed term)

**Related terms** 
Mesangial hypercellularity HP:0012574
Mesangial Immune complex deposition HP:0033295
Glomerular subepithelial immune-complex deposits HP:0033601
Systemic lupus erythematosus HP:0002725

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #152700, ORPHA:536

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
